src/components/Input/Input.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './Input.scss';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+function Input({
+  name, value, placeholder, isError, onChange, onFocus, errorMes,
+}) {
+  return (
+    <div className="wrapper">
+      <label htmlFor={name}>
+        <h2 className="fieldTitle">
+          {placeholder}
+          :
+        </h2>
+        <input
+          id={name}
+          value={value}
+          type="text"
+          name={name}
+          placeholder={placeholder.toLowerCase()}
+          className={classNames(
+            'textField', { redBorder: isError },
+          )}
+          onChange={event => onChange(event)}
+          onBlur={event => onFocus(event)}
+        />
+      </label>
+      <p
+        className={classNames(
+          'errorMes', { visible: isError },
```

Comment:
Don't use shortened words in names.

```suggestion
          'errorMessage', { visible: isError },
```